packages/headless/src/features/facets/category-facet-set/category-facet-set-actions.ts

```diff
@@ -53,7 +53,11 @@ export const registerCategoryFacet = createAction(
  */
 export const toggleSelectCategoryFacetValue = createAction(
   'categoryFacet/toggleSelectValue',
-  (payload: {facetId: string; selection: CategoryFacetValue}) => {
+  (payload: {
+    facetId: string;
+    selection: CategoryFacetValue;
+    numberOfValues: number;
```

Comment:
I find the name of this parameter a little confusing. Shouldn't it be called `retrieveCount` or `numberOfChildValues` (I prefer the former because it matches the search-api term)?

So, "select this value, and display a number of children equal to the retrieveCount"?